examples/deposit_erc20.rs

```diff
@@ -32,6 +42,14 @@ async fn main() -> Result<()> {
         .wallet(wallet)
         .on_http(l1_rpc_url);
 
+    let l1_gas_price = l1_provider.get_gas_price().await?;
+    // Deploy a test ERC20 token to be used for the deposit.
+    let erc20_token_address = ERC20Example::deploy_builder(&l1_provider)
+        .gas_price(l1_gas_price)
+        .deploy()
+        .await?;
+    println!("L1 ERC20 token address: {}", erc20_token_address);
```

Comment:
Sorry if stupid question, but shouldn't we also mint some tokens?

Reply:
It’s not needed, as this example token already mints tokens to the deployer. It’s the same token used in the Alloy example [here](https://alloy.rs/examples/transactions/transfer_erc20.html), and as you can see, there is no minting in this example either.